src/components/navigation/Navbar.vue

```diff
@@ -113,7 +113,7 @@ export default {
 .navbar {
   position: fixed;
   width: 100vw;
-  height: 55px;
+  // height: 55px;
```

Comment:
Explicit height is problematic. Height will grow to fit buttons. Control button sizing or padding to control height of navbar.